.gitignore

```
@@ -335,5 +335,13 @@ tracer/src/Datadog.Trace.ClrProfiler.Native/CMakeCache.txt
 tracer/src/Datadog.Trace.ClrProfiler.Native/cmake_install.cmake
 !tracer/src/Datadog.Trace.ClrProfiler.Native/lib/**
 
+# native sampleprofiler dependency build files
+tracer/test/test-applications/integrations/dependency-libs/Samples.AlwaysOnProfilerNativeDep/build/
+tracer/test/test-applications/integrations/dependency-libs/Samples.AlwaysOnProfilerNativeDep/Makefile
+tracer/test/test-applications/integrations/dependency-libs/Samples.AlwaysOnProfilerNativeDep/CMakeFiles/
+tracer/test/test-applications/integrations/dependency-libs/Samples.AlwaysOnProfilerNativeDep/cmake_install.cmake
+tracer/test/test-applications/integrations/dependency-libs/Samples.AlwaysOnProfilerNativeDep/CMakeCache.txt
+tracer/test/test-applications/integrations/dependency-libs/Samples.AlwaysOnProfilerNativeDep/out/
```

nit:

I think this could be changed to

```suggestion
tracer/test/test-applications/integrations/dependency-libs/Samples.AlwaysOnProfilerNativeDep/build/
Makefile
CMakeFiles
cmake_install.cmake
CMakeCache.txt
out
```

or (this I am not sure:
```suggestion
tracer/test/test-applications/integrations/dependency-libs/Samples.AlwaysOnProfilerNativeDep/build/
Makefile
CMakeFiles/
cmake_install.cmake
CMakeCache.txt
out/
```

---

I'd prefer to leave it as it is for now. `Cmake` related files from other native projects are excluded here already. If we decide to introduce additional `.gitignore` files, these other exclusions should be moved to dedicated `.gitignore` files as well.